packaging/test_irods_client_rest_cpp.py

```diff
@@ -278,7 +278,7 @@ def test_list(self):
     def test_list_honors_recursive__issue114():
         with session.make_session_for_existing_admin() as admin:
             try:
-                pwd, _ lib.
+                pwd, _ = lib.execute_command(['ipwd'])
```

Comment:
oh, i see.

hmm, let's find a way to use fullpaths - so we are more deterministic about where these tests do their work.

Reply:
Done